src/app/core/services/authorization.service.ts

```diff
@@ -1,28 +1,38 @@
 import { Injectable } from '@angular/core';
 import { Router } from '@angular/router';
-import { LoginForm } from '../models/authorization';
-import { User } from '../models/user';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
 
-const fakeUser: User = {
-  id: 1,
-  firstName: 'FirstName',
-  lastName: 'LastName',
-  email: '',
-};
+import { LoginForm, Token } from '../models/authorization';
+import { User } from '../models/user';
+import { catchError, Observable, tap, throwError } from 'rxjs';
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthorizationService {
+  private apiUrl = 'http://localhost:3004/auth';
   public isAuth: boolean = !!localStorage.getItem('accessToken');
-  constructor(private router: Router) {}
 
-  login(loginFormData: LoginForm): void {
-    fakeUser.email = loginFormData.email;
-    const userInfo = JSON.stringify(fakeUser);
-    localStorage.setItem('accessToken', 'fakeToken');
-    localStorage.setItem('userInfo', userInfo);
-    this.isAuth = true;
+  httpOptions = {
+    headers: new HttpHeaders({ 'Content-Type': 'application/json' }),
+  };
+
+  constructor(private router: Router, private http: HttpClient) {}
+
+  login(loginFormData: LoginForm): Observable<Token> {
+    return this.http
+      .post<Token>(`${this.apiUrl}/login`, loginFormData, this.httpOptions)
+      .pipe(
+        tap((data: Token) => {
+          localStorage.setItem('accessToken', data.token);
+          this.isAuth = true;
+        }),
+        catchError(() => {
+          return throwError(
```

Comment:
You can short callback func like this, no need to write retutrn:
` catchError(() => throwError(() => ...)`